git-gud.1

```diff
@@ -0,0 +1,68 @@
+.TH GIT-GUD 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
+.SH SYNOPSIS
+.B git gud
+[option1] [option2]
```

Comment:
I don't think this is actually the syntax we use. I'm thinking it might be better to take what git does:
```
SYNOPSIS
       git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>]
           <command> [<args>]
```

And do this:

```
SYNOPSIS
       git gud [--version] <command> [<args>]
```
or something similar

Reply:
I like it. I did not know the exact syntax, so I will make the relevant changes right now.